### ISSUE TITLE
Mongodb Support 

### ISSUE THREAD

**Issue by VikramSMalik** [goeh/grails-sequence-generator#9]
In this work with mongodb

**Comment by goeh** [comment]
I don't think it works with MongoDB as datasource. I have only tested it with SQL databases like H2, MySQL and SQL Server.
